Dockerfile

```diff
@@ -6,4 +6,6 @@ WORKDIR $PROJECT_ROOT
 COPY requirements.txt requirements.txt
 RUN pip install -r requirements.txt
 COPY . .
+COPY django-reference-app/django-reference-app/static/* /usr/local/lib/python3.6/site-packages/govuk_template_base/
```

Comment:
Manually copying files into `site-packages` is pretty unusual - unless this is some specific weirdness related to the govuk-template package, I'd guess that we should instead be configuring `settings.py` to serve static assets out of a `static` directory within the project.